tests/congruence_tests/test_common.py

```diff
@@ -68,6 +79,30 @@ def generate_fixtures(
         with_payload=True,
         random_ids=random_ids,
         skip_vectors=skip_vectors,
+        sparse=False,
+    )
+
+
+def generate_sparse_fixtures(
+        num: Optional[int] = NUM_VECTORS,
+        random_ids: bool = False,
+        vectors_sizes: Optional[Union[Dict[str, int], int]] = None,
+        skip_vectors: bool = False,
+        with_payload: bool = True,
+) -> List[models.Record]:
+    if vectors_sizes is None:
+        vectors_sizes = {
+            "sparse-text": text_vector_size,
+            "sparse-image": image_vector_size,
+            "sparse-code": code_vector_size,
```

Comment:
Does it not matter to use such small sizes?

Reply:
Sorry I don't get it. Could you please elaborate? :)

Reply:
Sparse vectors are supposed to be used for big "dimensionality", but these are pretty small (<= 100). Is that alright for the tests?